test/plots/waterfallPlotTests.ts

```diff
@@ -1,123 +1,251 @@
 ///<reference path="../testReference.ts" />
 
 describe("Plots", () => {
-  describe("Waterfall Plot", () => {
-    let svg: d3.Selection<void>;
-    let dataset: Plottable.Dataset;
-    let xScale: Plottable.Scales.Category;
-    let yScale: Plottable.Scales.Linear;
-    let plot: Plottable.Plots.Waterfall<string, number>;
-    let renderArea: d3.Selection<void>;
-    let SVG_WIDTH = 600;
-    let SVG_HEIGHT = 400;
-    let data = [
-      { x: "A", y: 20, t: "total" },
-      { x: "B", y: -5, t: "delta" },
-      { x: "C", y: 10, t: "delta" },
-      { x: "D", y: 25, t: "total" },
-      { x: "E", y: 5, t: "delta" },
-      { x: "F", y: -15, t: "delta" },
-      { x: "G", y: 15, t: "total" }
-    ];
-
-    beforeEach(() => {
-      svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      dataset = new Plottable.Dataset(data);
-      xScale = new Plottable.Scales.Category();
-      yScale = new Plottable.Scales.Linear();
-      plot = new Plottable.Plots.Waterfall<string, number>();
-      plot.x(function(d) { return d.x; }, xScale);
-      plot.y(function(d) { return d.y; }, yScale);
-      plot.total(function(d) { return d.t === "total" ? true : false; });
-      plot.addDataset(dataset);
-      plot.renderTo(svg);
-      renderArea = (<any> plot)._renderArea;
-    });
+  describe("Waterfall", () => {
+    describe("rendering growth bars", () => {
+      let numAttr = TestMethods.numAttr;
+      let svg: d3.Selection<void>;
+      let dataset: Plottable.Dataset;
+      let xScale: Plottable.Scales.Category;
+      let yScale: Plottable.Scales.Linear;
+      let plot: Plottable.Plots.Waterfall<string, number>;
+      let SVG_WIDTH = 600;
```

Comment:
this is not used anywhere else in the tests, does it have to be defined up there?
also, is there any reason for it to be 600 and 400? the default values won't work?